plugins/kotlin-dataframe/src/org/jetbrains/kotlinx/dataframe/plugin/impl/api/convert.kt

```diff
@@ -151,6 +158,20 @@ internal class To0 : AbstractInterpreter<PluginDataFrameSchema>() {
     }
 }
 
+internal class ConvertAsColumn : AbstractSchemaModificationInterpreter() {
+    val Arguments.receiver: ConvertApproximation by arg()
+    val Arguments.typeArg0: TypeApproximation by arg()
+    val Arguments.typeArg2: TypeApproximation by arg()
+    val Arguments.type: TypeApproximation by type(name("columnConverter"))
+
+    override fun Arguments.interpret(): PluginDataFrameSchema {
+        return receiver.schema.asDataFrame()
+            .replace { receiver.columns.map { it.toPath() }.toColumnSet() }
```

Comment:
it cannot call convert {}?

Reply:
Right, it can call convert asColumn directly. Why not :) Changed it